### PR TITLE
fixing something

### LINKED ISSUE #1: test issue 1
Some description here

### PR DESCRIPTION
**Pull request by SmilingWitch**
AAA `b` adasda. Close #1 

```typescript
function some()
```

- Item1
- Item2

### CODE REVIEW

File: file.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-function show2() {
+function show3() {
   console.log("some")
 }
 
-function show() {
+function show4() {
```
Comment (TheSnakeWitcher) [inline file.ts:5]: NO me gusto el nombre, cambia

File: file2.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function show2() {
+    console.log("some")
+  }
+  
+  function show() {
+    console.log("some2")
+  }
+
```
Comment (TheSnakeWitcher) [inline file2.ts:8]: Cambia el nombre tambien

Comment (SmilingWitch) [reply file2.ts]: Fixed in [d3e9e57](https://github.com/TheSnakeWitcher/nena-training/pull/3/commits/d3e9e57fa427f8c3790e4efbafa1d6ec117be040)

File: file.ts (hunk 1 above)
Comment (SmilingWitch) [reply file.ts]: Fixed in [11e0f5a](https://github.com/TheSnakeWitcher/nena-training/pull/3/commits/11e0f5a692963b1be9c5d96db0646ddd1da5ffbc)

**Pull request merged**